pytest_fixtures/core/contenthosts.py

```diff
@@ -221,3 +221,17 @@ def set_remote_execution_by_default(request, module_target_sat):
     )[0]
     remote_execution_setting.value = str(value)
     remote_execution_setting.update({'value'})
+
+
+@pytest.fixture
+def centos_host(request, version):
+    request.param = {"rhel_version": version.split('.')[0], "distro": "centos"}
+    with Broker(**host_conf(request), host_classes={'host': ContentHost}) as host:
+        yield host
+
+
+@pytest.fixture
+def oracle_host(request, version):
+    request.param = {"rhel_version": version.split('.')[0], "distro": "oracle"}
+    with Broker(**host_conf(request), host_classes={'host': ContentHost}) as host:
+        yield host
```

Comment:
I guess Convert2RHEL is not supported for 6.10, I'm unsure if this is required for 6.10.z, @shweta83 can confirm?

Reply:
Yes @Gauravtalreja1, we have designed test case for RHEL8 as well in `Convert2Rhel`  so it won't work on 6.10.

Reply:
Ok, Make sense ! Even I was confused about adding or removing this. Removing now !